charts/backstage/values.yaml

```diff
@@ -39,7 +46,40 @@ upstream:
         secretKeyRef:
           key: postgres-password
           name: "{{ .Release.Name }}-postgresql"
-
+    extraVolumeMounts:
+      - mountPath: /opt/app-root/src/dynamic-plugins-root
+        name: dynamic-plugins-root
+      - mountPath: /opt/app-root/src/dynamic-plugins-root/dynamic-plugins.yaml
+        name: dynamic-plugins
+        readOnly: true
+        subPath: dynamic-plugins.yaml
+    extraVolumes:
+      - emptyDir:
+          name: dynamic-plugins-root
+        name: dynamic-plugins-root
+      - configMap:
+          defaultMode: 420
+          name: dynamic-plugins
+          optional: true
+        name: dynamic-plugins
```

Comment:
No need for this if you use the suggestion made above.

Reply:
I don't see why. This config map doesn't contain a backstage `app-config` file, but the list of dynamic plugins used by the init container script to install the dynamic plugins and merge their `app-config` fragments to the additional `app-config.dynamic-plugins.yaml` file.